lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -78,11 +78,11 @@ sub remove_from_disk {
     my $file = $self->disk_file;
     OpenQA::Utils::log_debug("RM $file");
     if ($self->type eq 'iso') {
-        unlink($file);
+        unlink($file) || die "can't remove $file";
     }
     elsif ($self->type eq 'repo') {
         use File::Path qw(remove_tree);
-        remove_tree($file);
+        remove_tree($file) || die "can't remove $file";
```

Comment:
Wouldn't be warn sufficient? Or is it intentional to kill the gru? And will it kill it? (i.e. if the tasks are not evaled as timers are?)